app/grandchallenge/algorithms/templates/algorithms/algorithmmodel_form.html

```diff
@@ -3,7 +3,7 @@
 {% load url %}
 
 {% block title %}
-    {% if object %}Update{% else %}Create{% endif %} An Algorithm Model - {{ block.super }}
+    Create model - {{ algorithm.title }} - {{ block.super }}

```

Comment:
Indeed, it will be cleaner to use that. I am trying to stick with how it is done in the breadcrumbs. I prefer to. be consistent across all titles and use the `|yesno` construct. I will change and see later what James would say.

Reply:
Harmonzing all the breadcrumbs should follow (in a separate issue) after finishing the titles.